.travis/test_04_install.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+#
+# Copyright (c) 2018 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+export LC_ALL=C
+
+travis_retry docker pull "$DOCKER_NAME_TAG"
+env | grep -E '^(CCACHE_|WINEDEBUG|LC_ALL|BOOST_TEST_RANDOM|CONFIG_SHELL)' | tee /tmp/env
```

Comment:
It is somewhat confusing to set `LC_ALL=C.UTF-8` in the travis yaml, then set it to `LC_ALL=C` here, save it into a file, read that by docker, but then overwrite it again with `LC_ALL=C.UTF-8` for the test runner.

Reply:
The `lint-shell-locale.sh` requires scripts to set `LC_ALL=C`. I could change it to also accept `LC_ALL=C.UTF_8`.